CHANGELOG.txt

```diff
@@ -1,6 +1,7 @@
 7.x-1.14.1
 ----------
  - #2103 Fixed field mapping defaults for open data schema APIs
+ - Migration of CircleCI from V1 to V2
```

Comment:
Add the PR number (we will automate this some day)

Reply:
Done